common/changes/@microsoft/rush/fix-child-class-property_2022-12-09-05-51.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Fix LastLinkFlag path",
```

```suggestion
      "comment": "",
```

The issue hasn't been released yet.